### PR TITLE
feat(node,cli): allows 'edge' as possible runtime value

### PR DESCRIPTION
**Pull request by feugy**
### 📖 What's in there?

As part of Edge function GA, we're allowing `edge` as a runtime value.
This is adding to the allowed list, so we stay backward compatible.

[Linear ticket](https://linear.app/vercel/issue/EC-481/add-edge-to-the-list-of-allowed-runtimes)

I've kept one instance of `experimental-edge` in the test fixtures to ensure we still supports it.

####  🧪 Tests

- [x] The code changed/added as part of this PR has been covered with tests
- [x] All tests pass locally with `yarn test-unit`

#### Code Review

- [x] This PR has a concise title and thorough description useful to a reviewer
- [x] Issue from task tracker has a link to this PR

### CODE REVIEW

File: packages/node/src/dev-server.ts [hunk 1]
```diff
@@ -86,18 +86,17 @@ function listen(server: Server, port: number, host: string): Promise<void> {
...
-const validRuntimes = ['experimental-edge'];
 function parseRuntime(
   entrypoint: string,
   entryPointPath: string
 ): string | undefined {
   const project = new Project();
   const staticConfig = getConfig(project, entryPointPath);
   const runtime = staticConfig?.runtime;
-  if (runtime && !validRuntimes.includes(runtime)) {
+  if (runtime && !isEdgeRuntime(runtime)) {
```
Comment (Kikobeats) [inline packages/node/src/dev-server.ts:96]:
```suggestion
  if (!isEdgeRuntime(runtime)) {
```

**Review by Kikobeats — APPROVED** [review]
just dropping an unnecessary condition, the rest looks fine 🙂

**Review by TooTallNate — APPROVED** [review]

File: packages/node/src/dev-server.ts (hunk 1 above)
Comment (Kikobeats) [reply packages/node/src/dev-server.ts:96]: Since `isEdgeRuntime` is actually taking care about the nullish value

Comment (feugy) [reply packages/node/src/dev-server.ts:96]: That's a good point...in JS. Typescript complains about `includes()` not allowing `undefined` as a parameter:
image

**Pull request merged**